python/ray/serve/tests/test_fastapi.py

```diff
@@ -708,6 +708,9 @@ def decr2(self):
 
 
 @pytest.mark.parametrize("two_fastapi", [True, False])
+@pytest.mark.skip(
+    reason="Enable when https://github.com/ray-project/ray/issues/53024 is resolved"
```

Comment:
It will be enabled when https://github.com/ray-project/ray/pull/53647 is merged.